DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -1489,6 +1489,10 @@ private void GameBroadcastChannel_CTCPReceived(object sender, ChannelCTCPEventAr
                 if (cncnetGame == null)
                     return;
 
+                bool incompatible = cncnetGame == localGame && gameVersion != ProgramConstants.GAME_VERSION;
+                if (incompatible && ClientConfiguration.Instance.PreventIncompatibleGames)
+                    return;
```

Comment:
IMO would be better to do this at join level, as in add a message or a messagebox stating that this game is incompatible and you can't join it.

Reply:
I'm ok with that. We already have error messages that appear for other reasons. I'll add this one.

Reply:
Done.